frontend/__tests__/components/utils/datetime.spec.ts

```diff
@@ -33,36 +33,36 @@ describe('fromNow', () => {
       fromNow(new Date('Jan 01 1970 00:00:00'), new Date('Jan 01 1970 00:00:02'), {
         omitSuffix: true,
       }),
-    ).toEqual('few seconds');
+    ).toEqual('a few seconds');
```

Comment:
I don't think we need these tests any longer if we're just using the moment.js implementation. We're basically testing the moment.js code :)